plugin/converter/starlark/args.go

```diff
@@ -15,50 +15,98 @@
 package starlark
 
 import (
+	"fmt"
+
 	"github.com/drone/drone/core"
 
+	"reflect"
+
 	"go.starlark.net/starlark"
 	"go.starlark.net/starlarkstruct"
 )
 
-// TODO(bradrydzewski) add repository id
-// TODO(bradrydzewski) add repository timeout
-// TODO(bradrydzewski) add repository counter
-// TODO(bradrydzewski) add repository created
-// TODO(bradrydzewski) add repository updated
-// TODO(bradrydzewski) add repository synced
-// TODO(bradrydzewski) add repository version
-
-// TODO(bradrydzewski) add build id, will always be zero value
-// TODO(bradrydzewski) add build number, will always be zero value
-// TODO(bradrydzewski) add build started, will always be zero value
-// TODO(bradrydzewski) add build finished, will always be zero value
-// TODO(bradrydzewski) add build created, will always be zero value
-// TODO(bradrydzewski) add build updated, will always be zero value
-// TODO(bradrydzewski) add build parent
-// TODO(bradrydzewski) add build timestamp
+type GoSlice struct {
```

Comment:
what is this for?

Reply:
Good catch Marko, i forgot to delete this.